### ISSUE TITLE
Opinions please: Machine Access Changes, Solder Buff

### ISSUE THREAD

**Issue by GreatBigFailure** [d3athrow/vgstation13#29219]
Been thinking about adding this for a bit but it will likely be controversial so I'd like to get opinions before potentially wasting my time.

**Prior Rambling**
I find it odd that the robotics fabricator is the only machine of its type that has access restrictions, at least full restrictions (R&D has some, like removing materials and, for some reason, device linking). On the one hand it makes a roboticist/robotics access very precious if you actually want anything from the machine but on the other it stands out as a sore thumb and prevents casual use of it. 
Meaningwhile Cargo has a similar access restriction, though for a different type of machine, however you can remove that restriction by just soldering the board. That got me thinking, why not add that to the robotics fabricator? In fact, why not make more things work like that?

**The Actual Proposal**
What if we added/increased access restrictions on things like R&D consoles, engineering things, security cameras, etc. Then make it so solders can remove the restrictions; including existing restrictions with no current way to remove them.
It would make access more precious but also let people do whatever they need to on deadpop or whatever. It would also make greytiding a bit harder as well as leave more evidence. Secret buff to detectives. Also gives mechanics more of an actual job.

**tl;dr**
Add access restrictions to things. Soldering the circuitboard removes the access restrictions.

**Comment by DrSnips** [comment]
I think there was a pr in the past to make it "hackable" via panel/wires to unlock it like vendors. Which obviously didn't pass.

**Comment by Kurfursten** [comment]
I think hackable with wires would be more logically consistent

**Comment by Eneocho** [comment]
You could make it a hacking minigame like the jukebox's.

**Comment by GreatBigFailure** [comment]
The reason I go for the solder is because it's not included in most starting kits, sure it's easily accessible and so is the sulfuric it uses but it's at least harder than a multitool. It also forces you to take the machine apart which takes a lot longer than hacking does, even if you don't know the wire. 
Though wires do make more sense thematically, I think this works better mechanically. Maybe a mix?

**Comment by DrSnips** [comment]
I'm pretty sure the only reason the cargo console is done via solder is because it doesn't have wires to hack for access. Soldering for access changes is actually an oddity rather than the norm.

**Comment by Eneocho** [comment]
> 
> 
> The reason I go for the solder is because it's not included in most starting kits, sure it's easily accessible and so is the sulfuric it uses but it's at least harder than a multitool. It also forces you to take the machine apart which takes a lot longer than hacking does, even if you don't know the wire.
> Though wires do make more sense thematically, I think this works better mechanically. Maybe a mix?

Just give it a hacking pattern minigame like the jukebox. Say, it's 10 cables and you need to pulse 6 of them in a specific order to disable the access requirement. You'll be messing around for a while until you find the cables and the pattern.

**Comment by DrSnips** [comment]
As they say. Keep it simple. Just make it the usual pulse a wire to toggle access. Why make it over the top or some extra BS?

**Comment by GreatBigFailure** [comment]
Personally I do think the solder is a better choice because of how hard to be subtle with it it is. I have never had a scientist or doctor not freak out when I disassembled their machines. That's what this issue is for though, and collective opinion beats my own. 
Okay so wire hacking for machines and, for consistency, I assume still solder "hacking" for computers as @DrSnips said with the cargo computer? 

Now the access changes as a whole, yes or no? This wouldn't be a small change. I'd probably do it in order of increasing intensity. Restricting the protolathe, the cloner, DNA consoles, other major parts of a department that assistants/clowns should usually be asking to use rather than barging in for. Then progress to more controversial things like chem dispensers and autolathes if the less intrusive stuff goes over well. I have a feeling the latter won't be accepted though.

**Comment by DrSnips** [comment]
Don't cloners technically already have access locks save to start cloning, unless this is what you intend to add access locks to?

Considering stuff like vendors, particle accelerator, airlocks, safes, and such all use wires to hack for access I think that should be the route for machines.
Console based things can go solder because they lack wires.

As for the concept as a whole I have mixed feeling but currently I'm a bit more positive about it. On one hand it adds more value to picking a role but on the other people won't be entirely satisfied, even within their departments, because ~~they don't understand the concept of a role or~~ they need to fill the other parts of their departments. (See: parameds being whiny about surgery and chemistry access.) I can understand that people here hate to feel restricted so anything that impedes their ability to do literally anything they want is resisted.
Having a multitool (bonus if you have insulated gloves) is almost as powerful as having the spare itself. This then begs the question if access locks even matter if multitools are so easy to come by. And good luck passing the removal or limiting of insulated gloves and/or multitools.
Soldering iron is hardly a hard thing to come by either since they are from the same places as multitools with the exception of acid. But you can vend/ghetto chem enough from a soda vendor to hypothetically hack every machine in science currently, hack the engi vend for bottles, as well as just asking medical who will give you it 99% of the time no questions asked.

**Comment by west3436** [comment]
@GreatBigFailure implement this in a PR if you're happy with the feedback
